examples/patterns/aside.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Aside
+category: _patterns
+---
+
+<aside id="side-content" class="p-aside">
+  <div class="p-aside__section">
+    <h4 class="p-aside__header">Contents</h4>
+    <nav>
+      <ul class="p-toc">
```

Comment:
We should not use cross patterns in the examples as the user may not have `p-toc` and will be confused when this does not look like the example.